includes/class-mailchimp-woocommerce-service.php

```diff
@@ -640,20 +640,29 @@ public function setLandingSiteCookie()
         if ($this->getLandingSiteCookie() != false) return $this;
 
         $http_referer = $this->getReferer();
-
-        if (!empty($http_referer)) {
+        error_log("server". $_SERVER['REQUEST_URI'] );
```

Comment:
@alejandrogiv let's remove this log - or only if the "debugging" is set to true.